mobile/src/components/uam-groups-config/components/permissions-table/permissions-table.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import { useWindowDimensions } from 'react-native';
+
+import { PaginationDefaultValue } from '~/common/enums/enums';
+import { PermissionsGetAllItemResponseDto } from '~/common/types/types';
+import { Pagination, Table, View } from '~/components/common/common';
+import {
+  getPermissionsColumns,
+  getPermissionsRows,
+} from '~/components/uam-groups-config/helpers/helpers';
+import { useAppForm } from '~/hooks/hooks';
+
+type Props = {
+  permissions: PermissionsGetAllItemResponseDto[];
+  onCheckbox: (id: number) => void;
+  pagination: {
+    page: number;
+    setPage: (page: number) => void;
+  };
+  checkedPermissionsIds?: number[];
+};
+
+const PermissionsTable: FC<Props> = ({
+  permissions,
+  onCheckbox,
+  pagination,
+  checkedPermissionsIds,
+}) => {
+  const checkedIds: Record<string, boolean> = {};
+  const { control } = useAppForm({
+    defaultValues: checkedPermissionsIds?.length
+      ? checkedPermissionsIds.map(
+          (checkedId) => (checkedIds[`userIds.${checkedId}`] = true),
+        )
+      : {},
+  });
```

Comment:
We don't need this, we can control it via useSelectedItems outside, look at how it's done on frontend side